src/mlpack/methods/neighbor_search/sort_policies/nearest_neighbor_sort.hpp

```diff
@@ -150,6 +150,21 @@ class NearestNeighborSort
       return DBL_MAX;
     return a + b;
   }
+
+  /**
+   * Return the given value relaxed.
+   *
+   * @param value Value to relax.
+   * @param epsilon Relative error (non-negative).
+   *
+   * @return double Value relaxed.
+   */
+  static inline double Relax(const double value, const double epsilon)
+  {
+    if (value == DBL_MAX)
+      return DBL_MAX;
+    return (1 / (1 + epsilon)) * value;
```

Comment:
Shouldn't this be `(1 + epsilon)`? If I want to find an e=1.0-approximate furthest neighbor this means I am willing to accept a neighbor with half the distance from the query point as the true furthest neighbor. But this formula seems to imply that with e=1.0 I could get arbitrary amounts of approximation.